impeller/tessellator/tessellator.cc

```diff
@@ -54,19 +56,21 @@ VertexBuffer Tessellator::TessellateConvex(const Path& path,
   return VertexBuffer{
       .vertex_buffer = std::move(vertex_buffer),
       .index_buffer = std::move(index_buffer),
-      .vertex_count = index_buffer_->size(),
-      .index_type = IndexType::k16bit,
+      .vertex_count =
+          line_strip ? point_buffer_->size() : index_buffer_->size(),
+      .index_type = line_strip ? IndexType::kNone : IndexType::k16bit,
   };
 }
 
 void Tessellator::TessellateConvexInternal(const Path& path,
                                            std::vector<Point>& point_buffer,
                                            std::vector<uint16_t>& index_buffer,
-                                           Scalar tolerance) {
+                                           Scalar tolerance,
```

Comment:
Is tolerance used here? Is this just done for symmetry with the non-internal method?

Reply:
Its used on L75 below.

Reply:
🙈